### ISSUE TITLE
Add header filter support to persistence module

### ISSUE THREAD

**Issue by andimg93** [olifolkerd/tabulator#4091]
The persistence mode seems to work for all except the filtering. I tried
```javascript
persistence:{
	sort: true, //persist column sorting
	filter: true, //persist filter sorting
	group: true, //persist row grouping
	page: true, //persist page
	columns: true, //persist columns
}
```

and the default, which should activate all available persistence features:
```javascript
persistence: true
```

I talk about header based filtering => https://tabulator.info/docs/5.4/filter#header

**Comment by olifolkerd** [comment]
Please describe in detail what type of filtering you are expecting to persist

**Comment by andimg93** [comment]
well, as I linked - I mean the header based filtering, like in your documentation. 
1. Clicking on demo you go for an example.
2. As an example, based on that demo: If I enter now in the first column (name) "Oli", I get only the result "Oli Bob" and I would expect now that when I refresh the page and persistence mode/config is active that it simply takes into account / remembers the used / entered filters.

**Comment by olifolkerd** [comment]
Hey @olifolkerd 

Im afraid that header filters are not currently available in the persistence functionality, as stated in the docs:

> Note: Header filters are not currently stored persistently, this feature will be coming in a future release.

https://tabulator.info/docs/5.4/persist#filter

Cheers

Oli :)

**Comment by andimg93** [comment]
Oh I completely overlooked that, can you estimate when that will be approximately? That would be a really important feature

**Comment by olifolkerd** [comment]
@andimg93 not sure at the moment, hopefully one of the next few releases. 

Every feature request I receive is a "really important feature" for someone so I have to balance out the needs of the community against my time.

Will change this to a feature request and let you know when it is available

Cheers

Oli :)

**Comment by andimg93** [comment]
I absolutely understand, everybody has his own prio views - in any case many thanks for your effort!

**Comment by olifolkerd** [comment]
Hey All,

I have just merged this in to the 5.5 branch and will include it later in todays v5.5 release.

Cheers

Oli :)

**Comment by cgountanis** [comment]
Going to upgrade today, this is great news, been waiting for this feature.
